### PR TITLE
Task4359_addNewFeaturesCheckBoxes

### PR DESCRIPTION
**Pull request by MayaElf**
**Added:**
dense(y/n)
hint(y/n)
persistent-hint(text)
readonly(y/n)
error(y/n)
success(y/n)
background-color(material or css)

**NOTABENE** agreed that 'multiple' is default state of checkbox

### CODE REVIEW

File: components/form_inputs/checkboxes/VariousAttributesCheckbox.vue [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<template>
+  <v-container fluid>
+   <v-row align="center">
+     <span class="mr-3">Success, Readonly</span>
+         <p>{{ selected }}</p>
+             <v-checkbox
+               id="SuccessReadonlyCheckbox"
```
Comment (omega790) [inline components/form_inputs/checkboxes/VariousAttributesCheckbox.vue:7]: Please move id to header (span) - it'l be easier to find this element

File: components/form_inputs/checkboxes/VariousAttributesCheckbox.vue [hunk 2]
```diff
@@ -0,0 +1,58 @@
+<template>
+  <v-container fluid>
+   <v-row align="center">
```
Comment (omega790) [inline components/form_inputs/checkboxes/VariousAttributesCheckbox.vue:3]: Please add v-col into v-row

File: components/form_inputs/checkboxes/VariousAttributesCheckbox.vue [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+     <span class="mr-3">Success, Readonly</span>
+         <p>{{ selected }}</p>
+             <v-checkbox
+               id="SuccessReadonlyCheckbox"
+               v-model="selected"
+               label="John"
+               value="John"
+               success
```
Comment (omega790) [inline components/form_inputs/checkboxes/VariousAttributesCheckbox.vue:11]: I suppose you miss readonly property

File: components/form_inputs/checkboxes/VariousAttributesCheckbox.vue [hunk 4]
```diff
@@ -0,0 +1,58 @@
+<template>
+  <v-container fluid>
+   <v-row align="center">
+     <span class="mr-3">Success, Readonly</span>
+         <p>{{ selected }}</p>
```
Comment (omega790) [inline components/form_inputs/checkboxes/VariousAttributesCheckbox.vue:5]: Please delete this part as for this checkboxes we don't need it

File: components/form_inputs/checkboxes/VariousAttributesCheckbox.vue [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+               label="Jacob"
+               value="Jacob"
+               success
+             ></v-checkbox>
+   </v-row>
+   <v-row align="center">
+        <span class="mr-3">Error, Hint, Back-ground color, Dense</span>
+             <p>{{ selected }}</p>
+                 <v-checkbox
+                   id="ErrorDenseCheckboxWithHint"
```
Comment (omega790) [inline components/form_inputs/checkboxes/VariousAttributesCheckbox.vue:25]: Please move and this Id to span level

File: components/form_inputs/checkboxes/VariousAttributesCheckbox.vue [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+               id="SuccessReadonlyCheckbox"
+               v-model="selected"
+               label="Jacob"
+               value="Jacob"
+               success
+             ></v-checkbox>
+   </v-row>
+   <v-row align="center">
+        <span class="mr-3">Error, Hint, Back-ground color, Dense</span>
+             <p>{{ selected }}</p>
```
Comment (omega790) [inline components/form_inputs/checkboxes/VariousAttributesCheckbox.vue:23]: And delete this one

File: components/form_inputs/checkboxes/VariousAttributesCheckbox.vue (hunk 3 above)
Comment (MayaElf) [reply components/form_inputs/checkboxes/VariousAttributesCheckbox.vue:11]: done, thanks

**Review by omega790 — APPROVED** [review]

**Pull request merged**